t/node/healthcheck-passive.t

```diff
@@ -56,7 +56,7 @@ __DATA__
                                     "interval": 100,
                                     "http_failures": 2
                                 }
-                            },
+                            },]] .. [[
```

Comment:
Why do we need to do this? I'm confused.

Reply:
To work around the "too long code snippet" problem.

Reply:
got it, thx your explain